packages/tools/kadena-cli/src/prompts/wallets.ts

```diff
@@ -0,0 +1,105 @@
+import { program } from 'commander';
+
+import { getAllWallets } from '../keys/utils/keysHelpers.js';
+
+import { isValidFilename } from '../utils/helpers.js';
+import { log } from '../utils/logger.js';
+import { input, select } from '../utils/prompts.js';
+
+export async function walletNamePrompt(): Promise<string> {
+  return await input({
+    message: `Enter your wallet name:`,
+    validate: function (input) {
+      if (!isValidFilename(input)) {
+        return 'Wallet must be alphanumeric! Please enter a valid name.';
```

Comment:
We changed this to include other characters than only alphanumeric. 
Maybe we can do this? 

```suggestion
        return 'Name is used as a file name. Provide a valid filename (<add the regex>)';
```